azext_iot/tests/iothub/devices/test_hub_export_import.py

```diff
@@ -0,0 +1,328 @@
+import pytest
```

Comment:
tbh this file shouldn't be in devices since it touches more than just devices, move it to core or make a folder named state. Core folder is probably best option for now.